blockchain/chain.go

```diff
@@ -610,10 +610,12 @@ func (b *BlockChain) pruneBlockNodes() error {
 
 // isMajorityVersion determines if a previous number of blocks in the chain
 // starting with startNode are at least the minimum passed version.
-func (b *BlockChain) isMajorityVersion(minVer int32, startNode *blockNode, numRequired, numToCheck uint64) bool {
+func (b *BlockChain) isMajorityVersion(minVer int32, startNode *blockNode,
+	numRequired uint64) bool {
```

Comment:
Please either unwrap the function signature, or add a blank line after it to be consistent with the rest of the code.